packages/react/src/components/linkbox/Linkbox.module.scss

```diff
@@ -41,52 +39,72 @@
 }
 
 .contentSmall {
-  padding-left: 14px;
-  padding-right: 14px;
-  padding-bottom: 62px;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-bottom: 64px;
 }
 
 .contentMedium {
-  padding-left: 14px;
-  padding-right: 14px;
-  padding-bottom: 62px;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-bottom: 64px;
 }
 
 .contentLarge {
-  padding-left: 22px;
-  padding-right: 22px;
-  padding-bottom: 70px;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-bottom: 72px;
+}
+
+.withBorder {
+  .contentSmall {
+    padding-left: 14px;
+    padding-right: 14px;
+    padding-bottom: 62px;
+  }
+
+  .contentMedium {
+    padding-left: 14px;
+    padding-right: 14px;
+    padding-bottom: 62px;
+  }
+
+  .contentLarge {
+    padding-left: 22px;
+    padding-right: 22px;
+    padding-bottom: 70px;
+  }
+
+  .paddingWithoutImageAndWithBackground {
+    padding-top: 22px;
+  }
+}
+
+.contentLarge .text {
```

Comment:
Would it be possible to use the spacing tokens where they would make sense?
I know it is not possible always but for example where we are adding space between the border and the content they would make sense

Reply:
- [ ] Use spacing tokens